packages/bot-skeleton/src/constants/config.js

```diff
@@ -257,9 +257,10 @@ export const config = {
         [localize('Signal'), '2'],
     ],
     gd: {
-        cid: '828416594271-b4bhia944ecegn3j327oeb4l8o803bts.apps.googleusercontent.com',
-        aid: 'derivbot-248506',
-        api: 'AIzaSyA52MX2l8p75-w7nvab7fU6Lk6KwLqnyEI',
+        // Todo this change should revert after QA test
```

Comment:
Just making a comment here so we won't forget this. Please don't resolve this issue! 🙏